tools/etcd-dump-db/README.md

```diff
@@ -1,6 +1,10 @@
 ### etcd-dump-db
 
-etcd-dump-db inspects etcd db files.
+etcd-dump-db inspects etcd db files. Install and use the tool as described below.
+
+```
+  $ go get go.etcd.io/etcd/v3/tools/etcd-dump-db
```

Comment:
We need to use `go install `instead of `go get` to install a command. FYI. [go1.17#go-command](https://go.dev/doc/go1.17#go-command)

Reply:
@ahrtr thanks, makes sense. I am doing changing it for current instruction in the `benchmark` tool as well. Eventually we should separately update current usage of go get in code.